app/services/exports/lettings_log_export_service.rb

```diff
@@ -71,13 +72,12 @@ def get_archive_name(collection, base_number, increment)
       "core_#{collection}_#{collection + 1}_apr_mar_#{base_number_str}_#{increment_str}".downcase
     end
 
-    def write_export_archive(export, collection, start_time, full_update)
-      @logger.info("Writing export archives")
+    def write_export_archive(export, collection, start_time, recent_export, full_update)
       archive = get_archive_name(collection, export.base_number, export.increment_number) # archive name would be the same for all logs because they're already filtered by year (?)
 
       # Write archive
```

Comment:
```suggestion
      # Create archive
```

Reply:
I mean I'll remove it fully, cause it's literally in the method name 😅